services/drupal/web/modules/custom/epa_migrations/src/Plugin/migrate/source/EpaFile.php

```diff
@@ -38,6 +38,23 @@ protected function initializeIterator() {
     return SqlBase::initializeIterator();
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function query() {
+    $query = parent::query();
+
+    // Don't use timestamp highwater because it isn't unique, and stops/restarts
+    // can result in files being skipped. Remove the parent's sort and replace
+    // with fid.
+    $sort =& $query->getOrderBy();
+    unset($sort['f.timestamp']);
+    // Use fid highwater mark
+    $query->orderBy('f.fid');
```

Comment:
This is not necessary -- there's other code that handles adding the orderBy for the high water mark.  We only need to remove the timestamp sort.